{
  "active": false,
  "connections": {
    "Error Trigger": {
      "main": [
        [
          {
            "node": "Discord",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "createdAt": "2025-02-04T17:37:16.587Z",
  "id": "B5ErT6ObkzMJxzKU",
  "meta": {
    "templateCredsSetupCompleted": true
  },
  "name": "‚öôÔ∏è Aviso de Erro no Discord",
  "nodes": [
    {
      "parameters": {},
      "id": "595ec15d-638f-4fc0-955e-a872e7c4e721",
      "name": "Error Trigger",
      "type": "n8n-nodes-base.errorTrigger",
      "typeVersion": 1,
      "position": [
        140,
        -80
      ]
    },
    {
      "parameters": {
        "webhookUri": "https://discord.com/api/webhooks/1327342713888178176/F6VjC83THfIemEVxo_22VN-_reCyT3NZmaU1wASXw1fLrzMm44TUVQCKDLtlv63jQF7h",
        "text": "=üö® **Workflow com erro no N8N da Portus Externo**\n\nO Workflow **{{ $json[\"workflow\"][\"name\"] }}**, do **ID {{ $json[\"workflow\"][\"id\"] }}**, acabou de apresentar um erro.\n\nAssim que puder, d√™ uma conferida e ajuste o erro apresentado.",
        "options": {}
      },
      "id": "84a4122c-8804-47e2-8337-6637635b2ee9",
      "name": "Discord",
      "type": "n8n-nodes-base.discord",
      "typeVersion": 1,
      "position": [
        360,
        -80
      ]
    }
  ],
  "pinData": {},
  "settings": {
    "executionOrder": "v1"
  },
  "staticData": null,
  "tags": [],
  "triggerCount": 0,
  "updatedAt": "2025-02-04T17:37:44.103Z",
  "versionId": "7910a0eb-30ca-438f-a925-760f79f2ba42"
}